### PR TITLE
fixed absolute and msdn links

### LINKED ISSUE #756: Links to other core-docs topics should be relative
We have a mix of relative and absolute links, even when linking to topics that are part of the same doc site.  

Where the target document is resident in the core-docs repository, we should use relative links. It will facilitate better testing in our local development environment.

/cc @svick

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #756 

/cc @BillWagner @cartermp 

Removed en-us and version from msdn links so that localized versions in the future will take the user to the language stored in their browser cache. Removing the version to always go to the tip.
Made other small fixes on the affected topics.

I haven't changed the API links that will be handled separately and are tracked by this issue: #787

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.reflection.targetinvocationexception?branch=pr-en-us-1001)

File: docs/fsharp/language-reference/import-declarations-the-open-keyword.md [hunk 1]
```diff
@@ -61,11 +61,11 @@ Some namespaces are so frequently used in F# code that they are opened implicitl
 |`Microsoft.FSharp.Text`|Contains functions for formatted IO, such as the `printf` function.|
 
 ## AutoOpen Attribute
-You can apply the `AutoOpen` attribute to an assembly if you want to automatically open a namespace or module when the assembly is referenced. You can also apply the `AutoOpen` attribute to a module to automatically open that module when the parent module or namespace is opened. For more information, see [Core.AutoOpenAttribute Class](https://stage.docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/core.autoopenattribute-class).
+You can apply the `AutoOpen` attribute to an assembly if you want to automatically open a namespace or module when the assembly is referenced. You can also apply the `AutoOpen` attribute to a module to automatically open that module when the parent module or namespace is opened. For more information, see [Core.AutoOpenAttribute Class](https://msdn.microsoft.com/visualfsharpdocs/conceptual/core.autoopenattribute-class-%5bfsharp%5d).
```
Comment (BillWagner) [inline docs/fsharp/language-reference/import-declarations-the-open-keyword.md]: Why dod you move from a docs.microsoft.com article to an MSDN article?

File: docs/fsharp/language-reference/import-declarations-the-open-keyword.md [hunk 2]
```diff
@@ -61,11 +61,11 @@ Some namespaces are so frequently used in F# code that they are opened implicitl
...
+Some modules, records, or union types may specify the `RequireQualifiedAccess` attribute. When you reference elements of those modules, records, or unions, you must use a qualified name regardless of whether you include an import declaration. If you use this attribute strategically on types that define commonly used names, you help avoid name collisions and thereby make code more resilient to changes in libraries. For more information, see [Core.RequireQualifiedAccessAttribute Class](https://msdn.microsoft.com/visualfsharpdocs/conceptual/core.requirequalifiedaccessattribute-class-%5Bfsharp%5D).
```
Comment (BillWagner) [inline docs/fsharp/language-reference/import-declarations-the-open-keyword.md]: Same question here: moving from docs.micrsoft.com to MSDN

File: docs/fsharp/tutorials/type-providers/accessing-a-web-service.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ type TerraService = WsdlService<" HYPERLINK "http://terraserver-usa.com/TerraSer
...
+<br />  There are two ways to specify configuration settings for a WSDL connection, by using the app.config file for the project, or by using the static type parameters in the type provider declaration. You can use svcutil.exe to generate appropriate configuration file elements. For more information about using svcutil.exe to generate configuration information for a web service, see [ServiceModel Metadata Utility Tool &#40;Svcutil.exe&#41;](https://msdn.microsoft.com/library/aa347733.aspx).For a full description of the static type parameters for the WSDL type provider, see [WsdlService Type Provider](https://msdn.microsoft.com/visualfsharpdocs/conceptual/wsdlservice-type-provider-%5bfsharp%5d).
```
Comment (BillWagner) [inline docs/fsharp/tutorials/type-providers/accessing-a-web-service.md]: There should be a space after the full stop and before "For a full description...."

**Comment by BillWagner** [comment]
Looks great. I noticed two small typos, and had a couple questions.

Once you fix the typos, :shipit:

File: docs/fsharp/language-reference/import-declarations-the-open-keyword.md (hunk 1 above)
Comment (svick) [inline docs/fsharp/language-reference/import-declarations-the-open-keyword.md]: I don't think stage.docs.microsoft.com should appear in published docs (the old link certainly doesn't work for me).

I'm assuming it's related to this note on [another F# article](https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/symbol-and-operator-reference/index):

> The API reference links in this article will take you to MSDN. The docs.microsoft.com API reference is not complete.

Comment (cartermp) [inline docs/fsharp/language-reference/import-declarations-the-open-keyword.md]: Yep, that's why.

**Comment by cartermp** [comment]
LGTM!

File: docs/fsharp/language-reference/import-declarations-the-open-keyword.md (hunk 1 above)
Comment (mairaw) [inline docs/fsharp/language-reference/import-declarations-the-open-keyword.md]: Yep, this topic hasn't been moved to docs yet and I talked to @cartermp offline to confirm the change before I submitted this PR.

File: docs/fsharp/tutorials/type-providers/accessing-a-web-service.md (hunk 3 above)
Comment (mairaw) [inline docs/fsharp/tutorials/type-providers/accessing-a-web-service.md]: Good catch. I just replaced the links. I'll fix this.

**Comment by mairaw** [comment]
@BillWagner I saw the comment about the missing space. Was there another typo (you mention two in your comment)? The change to MSDN links was on purpose.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.linq.expressions.membermemberbinding?branch=pr-en-us-1001)

**Comment by BillWagner** [comment]
@mairaw It was just the one.  Off-by-one error on my part.

**Comment by mairaw** [comment]
No problem @BillWagner. Just wanted to make sure I didn't miss anything. I'll fix the conflicts and merge this. Thanks!

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.runtime.serialization.iformatterconverter?branch=pr-en-us-1001)

File: docs/fsharp/tutorials/type-providers/accessing-a-web-service.md [hunk 4]
```diff
@@ -79,7 +79,7 @@ type TerraService = WsdlService<" HYPERLINK "http://terraserver-usa.com/TerraSer
...
+<br />  There are two ways to specify configuration settings for a WSDL connection, by using the app.config file for the project, or by using the static type parameters in the type provider declaration. You can use svcutil.exe to generate appropriate configuration file elements. For more information about using svcutil.exe to generate configuration information for a web service, see [ServiceModel Metadata Utility Tool &#40;Svcutil.exe&#41;](https://msdn.microsoft.com/library/aa347733.aspx). For a full description of the static type parameters for the WSDL type provider, see [WsdlService Type Provider](https://msdn.microsoft.com/visualfsharpdocs/conceptual/wsdlservice-type-provider-%5bfsharp%5d).
 <br />
```
Comment (mairaw) [inline docs/fsharp/tutorials/type-providers/accessing-a-web-service.md]: @cartermp I've noticed that we use some HTML elements throughout the F# content. Some are needed like the one for the pipes inside tables but in some cases they seem unnecessary like the <br> tags in this topic. Should we open a low-pri issue to clean HTML tags throughout the content?

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some sample preview links:
- Docset 1: [core-docs](https://stage.docs.microsoft.com/en-us/dotnet/core/api/system.reflection.emit.parameterbuilder?branch=pr-en-us-1001)

**Pull request merged**